SchemaDatabase/SGr/Generic/BaseTypes.xsd

```diff
@@ -454,199 +454,199 @@ https://github.com/SmartgridReady/SGrSpecifications  -->
-      <enumeration value="KILOGRAMS_PER_HOUR" />
-      <enumeration value="KILOGRAMS_PER_MINUTE" />
-      <enumeration value="KILOGRAMS_PER_SECOND" />
-      <enumeration value="KILOHERTZ" />
-      <enumeration value="KILOOHMS" />
-      <enumeration value="KILOJOULES" />
-      <enumeration value="KILOJOULES_PER_DEGREE_KELVIN" />
-      <enumeration value="KILOJOULES_PER_KILOGRAM" />
-      <enumeration value="KILOJOULES_PER_KILOGRAM_DRY_AIR" />
-      <enumeration value="KILOMETERS_PER_HOUR" />
-      <enumeration value="KILOPASCALS" />
-      <enumeration value="KILOVOLT_AMPERES" />
-      <enumeration value="KILOVOLT_AMPERE_HOURS" />
-      <enumeration value="KILOVOLT_AMPERES_REACTIVE" />
-      <enumeration value="KILOVOLT_AMPERES_REACTIVE_HOURS" />
-      <enumeration value="KILOVOLTS" />
+      <!-- <enumeration value="KILOGRAMS_PER_CUBIC_METER" /> -->
+      <!-- <enumeration value="KILOGRAMS_PER_HOUR" /> -->
+      <!-- <enumeration value="KILOGRAMS_PER_MINUTE" /> -->
+      <!-- <enumeration value="KILOGRAMS_PER_SECOND" /> -->
+      <!-- <enumeration value="KILOHERTZ" /> -->
+      <!-- <enumeration value="KILOOHMS" /> -->
+      <!-- <enumeration value="KILOJOULES" /> -->
+      <!-- <enumeration value="KILOJOULES_PER_DEGREE_KELVIN" /> -->
+      <!-- <enumeration value="KILOJOULES_PER_KILOGRAM" /> -->
+      <!-- <enumeration value="KILOJOULES_PER_KILOGRAM_DRY_AIR" /> -->
+      <!-- <enumeration value="KILOMETERS_PER_HOUR" /> -->
+      <!-- <enumeration value="KILOPASCALS" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERES" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERE_HOURS" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERES_REACTIVE" /> -->
+      <!-- <enumeration value="KILOVOLT_AMPERES_REACTIVE_HOURS" /> -->
+      <!-- <enumeration value="KILOVOLTS" /> -->
       <enumeration value="KILOWATT_HOURS" />
-      <enumeration value="KILOWATTS" />
-      <enumeration value="KW_HOURS_PER_SQUARE_FOOT" />
-      <enumeration value="KW_HOURS_PER_SQUARE_METER" />
-      <enumeration value="LITERS" />
-      <enumeration value="LITERS_PER_HOUR" />
-      <enumeration value="LITERS_PER_MINUTE" />
-      <enumeration value="LITERS_PER_SECOND" />
-      <enumeration value="LUMENS" />
-      <enumeration value="LUXES" />
-      <enumeration value="MEGA_BTUS" />
-      <enumeration value="MEGAHERTZ" />
-      <enumeration value="MEGAJOULES" />
-      <enumeration value="MEGAJOULES_PER_DEGREE_KELVIN" />
-      <enumeration value="MEGAJOULES_PER_KILOGRAM_DRY_AIR" />
-      <enumeration value="MEGAJOULES_PER_SQUARE_FOOT" />
-      <enumeration value="MEGAJOULES_PER_SQUARE_METER" />
-      <enumeration value="MEGAVOLT_AMPERES" />
-      <enumeration value="MEGAVOLT_AMPERES_REACTIVE" />
-      <enumeration value="MEGAVOLTS" />
-      <enumeration value="MEGAWATT_HOURS" />
-      <enumeration value="MEGAWATTS" />
-      <enumeration value="MEGOHMS" />
+      <!-- <enumeration value="KILOWATTS" /> -->
```

Comment:
`KILOWATTS` wird auch oft und überall benutzt - auch in Published FP's - ändern in `WATTS`?

Reply:
wir können KILOWATTS auskommentieren